ui/src/App.tsx

```diff
@@ -8,6 +8,15 @@ import '@agoric/react-components/dist/style.css';
 
 function App() {
   const { themeClass } = useTheme();
+  let REST_HOSTNAME = 'http://localhost:1317';
+  let RPC_HOSTNAME = 'http://localhost:26657';
+
+  const codeSpaceHostName = import.meta.env.VITE_HOSTNAME;
+
+  if (codeSpaceHostName) {
+    REST_HOSTNAME = `https://${codeSpaceHostName}-1317.app.github.dev/`;
```

Comment:
Please remove extra `/` at the end of new URLs to be consistent with localhost hostname above.